lib/cloudhsm/cloudhsm_key_generator.rb

```diff
@@ -0,0 +1,70 @@
+require 'greenletters'
```

Comment:
I know this is basically a oneoff, but this whole file could use more comments describing what it is, what it does, and how. For example if I ran across this file I would have no idea that the primary interface to it is actually through the rake task.